scenarios/test_codegen/test/lib_tests/FetchState_test.res

```diff
@@ -2482,3 +2482,28 @@ describe("Test queue item", () => {
     )
   })
 })
+
+describe("FetchState.queueItemIsInReorgThreshold", () => {
+  it("Returns false when we just started the indexer and it has currentBlockHeight=0", () => {
+    let fetchState = makeInitial()
+    Assert.equal(
+      fetchState
+      ->FetchState.getEarliestEvent
+      ->FetchState.queueItemIsInReorgThreshold(~heighestBlockBelowThreshold=0),
+      false,
+    )
+  })
+
+  it(
+    "Returns false when we just started the indexer and it has currentBlockHeight=0, while start block is more than 0 + reorg threshold",
```

Comment:
This test was returning true

Reply:
Was the first test passing the assertion?

Reply:
Yes